src/Native/System.Native/pal_process.cpp

```diff
@@ -53,6 +54,11 @@ static_assert(PAL_LOG_LOCAL5 == LOG_LOCAL5, "");
 static_assert(PAL_LOG_LOCAL6 == LOG_LOCAL6, "");
 static_assert(PAL_LOG_LOCAL7 == LOG_LOCAL7, "");
 
+// Validate that out PriorityWhich values are correct for the platform
+static_assert(static_cast<int>(PriorityWhich::PAL_PRIO_PROCESS) == PRIO_PROCESS, "");
```

Comment:
I made our enums not have the 'class' designation so that we can get rid of a bunch of casts like this: i.e. enum Foo : T instead of enum class Foo : T. Please fix the new ones up to match.